styles/style.css

```diff
@@ -88,33 +103,44 @@ body {
 	max-width: 100px;
 	background: #ce5f31;
 }
-
+.container {
+	flex: 1 1 30%;
+}
+.box {
+  height: 150px;
+  width: 150px;
+  background-color: lightgray;
+  font-size: 1.2rem;
+  font-weight: bold;
+  text-align: center;
+  margin: 1% auto 3% auto;
+  padding: 1%;
+}
 /* Article */
 .article {
 	margin-bottom: 2em;
+	border: 1px solid gray;
+	background-color: #D8FFD6;
+	margin:1% 1%;
```

Comment:
Same thing, you use the `article` class on the same level as `col-md-5`: `article` defines its own margin which will cause problems with the grid. Also, you are putting the border/background-color directly on the `col-` element, which will render unexpectedly. Bootstrap is for invisible boxes - if you want the box to be visible, that needs to happen on a child element inside the column.